### ISSUE TITLE
About Screen Weirdness

### ISSUE THREAD

**Issue by YouveGotMeowxy** [stevencohn/OneMore#795]
**Describe the bug**

Several weird things happen when using the about screen; examples:

- Nothing happens when clicking link in About screen + wont show again: I clicked the About screen, then clicked the Check for Update link. Saw nothing happen; after that clicking the same About screen from the OneMore menu doesn't bring anything up (no about screen at all).

- Clicking the Update link crashes OM:

<details>
  <summary>Log</summary>

  ```
  See the end of this message for details on invoking 
just-in-time (JIT) debugging instead of this dialog box.

************** Exception Text **************
System.NullReferenceException: Object reference not set to an instance of an object.
   at River.OneMoreAddIn.OneNote.GetCurrentMainWindowBounds()
   at River.OneMoreAddIn.UI.LocalizableForm.OnLoad(EventArgs e)
   at System.Windows.Forms.Form.OnCreateControl()
   at System.Windows.Forms.Control.CreateControl(Boolean fIgnoreVisible)
   at System.Windows.Forms.Control.CreateControl()
   at System.Windows.Forms.Control.WmShowWindow(Message& m)
   at System.Windows.Forms.Control.WndProc(Message& m)
   at System.Windows.Forms.Form.WmShowWindow(Message& m)
   at System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)


************** Loaded Assemblies **************
mscorlib
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.9105.0 built by: NET481REL1LAST_C
    CodeBase: file:///C:/Windows/Microsoft.NET/Framework64/v4.0.30319/mscorlib.dll
----------------------------------------
River.OneMoreAddIn
    Assembly Version: 5.4.1.0
    Win32 Version: 5.4.1
    CodeBase: file:///C:/Program%20Files/OneMoreAddIn/River.OneMoreAddIn.dll
----------------------------------------
office
    Assembly Version: 15.0.0.0
    Win32 Version: 15.0.4613.1000
    CodeBase: file:///C:/Windows/assembly/GAC_MSIL/office/15.0.0.0__71e9bce111e9429c/office.dll
----------------------------------------
System.Drawing
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.9037.0 built by: NET481REL1
    CodeBase: file:///C:/Windows/Microsoft.Net/assembly/GAC_MSIL/System.Drawing/v4.0_4.0.0.0__b03f5f7f11d50a3a/System.Drawing.dll
----------------------------------------
System
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.9093.0 built by: NET481REL1LAST_C
    CodeBase: file:///C:/Windows/Microsoft.Net/assembly/GAC_MSIL/System/v4.0_4.0.0.0__b77a5c561934e089/System.dll
----------------------------------------
System.Windows.Forms
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.9093.0 built by: NET481REL1LAST_C
    CodeBase: file:///C:/Windows/Microsoft.Net/assembly/GAC_MSIL/System.Windows.Forms/v4.0_4.0.0.0__b77a5c561934e089/System.Windows.Forms.dll
----------------------------------------
System.Xml.Linq
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.9037.0 built by: NET481REL1
    CodeBase: file:///C:/Windows/Microsoft.Net/assembly/GAC_MSIL/System.Xml.Linq/v4.0_4.0.0.0__b77a5c561934e089/System.Xml.Linq.dll
----------------------------------------
System.Core
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.9105.0 built by: NET481REL1LAST_C
    CodeBase: file:///C:/Windows/Microsoft.Net/assembly/GAC_MSIL/System.Core/v4.0_4.0.0.0__b77a5c561934e089/System.Core.dll
----------------------------------------
System.Xml
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.9037.0 built by: NET481REL1
    CodeBase: file:///C:/Windows/Microsoft.Net/assembly/GAC_MSIL/System.Xml/v4.0_4.0.0.0__b77a5c561934e089/System.Xml.dll
----------------------------------------
System.Management
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.9037.0 built by: NET481REL1
    CodeBase: file:///C:/Windows/Microsoft.Net/assembly/GAC_MSIL/System.Management/v4.0_4.0.0.0__b03f5f7f11d50a3a/System.Management.dll
----------------------------------------
System.Web.Extensions
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.9037.0
    CodeBase: file:///C:/Windows/Microsoft.Net/assembly/GAC_MSIL/System.Web.Extensions/v4.0_4.0.0.0__31bf3856ad364e35/System.Web.Extensions.dll
----------------------------------------
System.Web
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.9037.0 built by: NET481REL1
    CodeBase: file:///C:/Windows/Microsoft.Net/assembly/GAC_64/System.Web/v4.0_4.0.0.0__b03f5f7f11d50a3a/System.Web.dll
----------------------------------------
System.Configuration
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.9037.0 built by: NET481REL1
    CodeBase: file:///C:/Windows/Mi
  ```

</details>

- Clicking the GH link takes a LONG time, and then after about2-3 minutes, loads the page. In the meantime, the About screen title changes to "not responding" with the hourglass icon.
- After clicking the About item, and the about screen displays, OneNote itself is unusable, and the About screen "disappears"; even when manually minimizing everything 1 by one. Only way to get it back was to use the Show Desktop thing on the taskbar and then clicking OneNote again.

**Environment (if applicable):**
 - OneNote Version: 

![image](https://user-images.githubusercontent.com/39491297/206827024-537c24a4-27c1-4d0e-bc93-58ae3e2d2b11.png)

 - OneMore Version

![image](https://user-images.githubusercontent.com/39491297/206827015-00c76e28-9cf7-4291-95e2-ff4ada183f7c.png)

**Comment by stevencohn** [comment]
@YouveGotMeowxy Sorry you're experiencing issues. How is your network connectivity otherwise? Clicking either the github link or the Check for updates link are very simple requests to the OS to open their respective URLs.

The GetCurrentMainWindowBounds issue, which is a secondary effect, will be fixed by #790

**Comment by YouveGotMeowxy** [comment]
> @YouveGotMeowxy Sorry you're experiencing issues. How is your network connectivity otherwise? Clicking either the github link or the Check for updates link are very simple requests to the OS to open their respective URLs.

Hmmm, generally speaking my internet connection is working fine, even when that happens. I guess I'll just ignore it for now. Just thought I'd put in a report in case no one was aware, and/or others that make experience it can check in. :)
